src/modules/nats/nats-container.test.ts

```diff
@@ -0,0 +1,75 @@
+import { connect, StringCodec } from "nats";
+import { NatsContainer } from "./nats-container";
+
+describe("NatsContainer", () => {
+  jest.setTimeout(180_000);
+
+  it("should start, connect and close", async () => {
+    const container = await new NatsContainer().start();
+
+    // establish connection
+    const nc = await connect(container.getConnectionOptions());
+    // close the connection
+    await nc.close();
+    // check if the close was OK
+    const err = await nc.closed();
+    expect(err).toBe(undefined);
+
+    await container.stop();
+  });
+
+  it("should subscribe and receive one published message", async () => {
+    const SUBJECT = "HELLO";
+    const PAYLOAD = "WORLD";
+
+    const container = await new NatsContainer().start();
+    const nc = await connect(container.getConnectionOptions());
+    const sc = StringCodec();
+
+    //----------------
+    const sub = nc.subscribe(SUBJECT);
+    (async () => {
```

Comment:
Is the IIFE required/can the console.log be removed?

Reply:
can be removed